__init__.py

```diff
@@ -267,10 +272,27 @@ def initialize(self):
         self.gui.register_handler("mycroft.return.select.backend", self.handle_return_event)
         self.gui.register_handler("mycroft.device.confirm.stt", self.handle_stt_selected)
         self.gui.register_handler("mycroft.device.confirm.tts", self.handle_tts_selected)
+        self.gui.register_handler("mycroft.device.confirm.language", self.handle_language_selected)
+        self.gui.register_handler("mycroft.return.select.language", self.handle_language_back_event)
         self.nato_dict = self.translate_namedvalues('codes')
 
         self._init_state()
 
+        if "enable_language_support" not in self.settings:
```

Comment:
since you are working on this, can you add these sections for all steps?

 "enable_stt_selection", "enable_tts_selection", "enable_backend_selection" ?
 
 this has been requested and was on the roadmap, but since you are doing it for langs maybe you can do it for all steps

Reply:
ill make the PR

Reply:
please rename to "enable_language_selection" instead of support for consistency

Reply:
fixed